src/layers/domain/core/tests/test_ods_organisation.py

```diff
@@ -1,34 +1,34 @@
 import pytest
-from domain.core.error import InvalidOdsCodeError
-from domain.core.product import OdsOrganisation
+from domain.core.ods_organisation import OdsOrganisation
 from domain.core.root import Root
+from pydantic import ValidationError
 
 
 @pytest.mark.slow
```

Comment:
can remove the `slow` marker from these non-slow tests that don't use the ODS api anymore